src/Transports.Subscriptions.WebSockets/WebSocketTransport.cs

```diff
@@ -3,26 +3,18 @@
 using System.Threading;
 using System.Threading.Tasks;
 using GraphQL.Server.Transports.Subscriptions.Abstractions;
-using Newtonsoft.Json;
-using Newtonsoft.Json.Serialization;
 
 namespace GraphQL.Server.Transports.WebSockets
 {
     public class WebSocketTransport : IMessageTransport, IDisposable
     {
         private readonly WebSocket _socket;
 
-        public WebSocketTransport(WebSocket socket, IGraphQLSerializer serializer)
+        public WebSocketTransport(WebSocket socket, IGraphQLTextSerializer serializer)
         {
             _socket = socket;
-            var serializerSettings = new JsonSerializerSettings
-            {
-                DateFormatHandling = DateFormatHandling.IsoDateFormat,
-                DateFormatString = "yyyy'-'MM'-'dd'T'HH':'mm':'ss.FFFFFFF'Z'",
-                ContractResolver = new CamelCasePropertyNamesContractResolver()
-            };
```

Comment:
GraphQL.NET formats dates as strings internally now, so this code doesn't matter.

Reply:
ok